src/components/standalone/backup_and_restore/DrawerSetPassphrase.vue

```diff
@@ -0,0 +1,122 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { ubusCall } from '@/lib/standalone/ubus'
+import {
+  getAxiosErrorMessage,
+  NeButton,
+  NeInlineNotification,
+  NeSideDrawer,
+  NeTextInput,
+  NeTitle,
+  NeTooltip
+} from '@nethserver/vue-tailwind-lib'
+import { AxiosError } from 'axios'
+
+const { t } = useI18n()
+
+const props = defineProps({
+  showPassphraseDrawer: {
+    type: Boolean,
+    required: true
+  },
+  unitName: {
+    type: String
+  }
+})
+
+const emit = defineEmits(['success', 'close'])
+
+watch(
+  () => props.showPassphraseDrawer,
+  () => {
+    errorSetPassphrase.value = {
+      notificationTitle: '',
+      notificationDescription: '',
+      passphrase: ''
+    }
+  }
+)
+
+const formPassphrase = ref({
+  passphrase: ''
+})
+
+let loadingSetPassphrase = ref(false)
+let passphraseRef = ref()
+
+let objNotification = {
+  notificationTitle: '',
+  notificationDescription: '',
+  passphrase: ''
+}
+
+let errorSetPassphrase = ref({ ...objNotification })
```

Comment:
Same as other file, avoid this kind of approach, object interpolation in JS is no joke (sadly)
If you need this for an hypothetical error, just do something like this
```suggestion
interface CustomError = {
  title: string,
  description: string
}

const errorSetPassphrase = ref<CustomError>()
```

then check if the variable is defined to show something, or set to undefined to hide all